.github/workflows/main_workflow.yml

```diff
@@ -164,7 +164,7 @@ jobs:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
         python-version: [ "3.7", "3.8", "3.9", "3.10" ]
 
-    timeout-minutes: 65
+    timeout-minutes: 70
```

Comment:
Increased timeout as I have noticed windows tests timing out lately and I also added two new `hypothesis` tests.